torch/onnx/_internal/registration.py

```diff
@@ -0,0 +1,307 @@
+"""Module for handling symbolic function registration."""
+
+import functools
+import importlib
+import inspect
+import itertools
+import warnings
+from typing import Callable, Collection, Dict, Generic, Optional, Set, TypeVar
+
+from torch.onnx import _constants, errors
+
+OpsetVersion = int
+
+
+def _dispatch_opset_version(
+    target: OpsetVersion, available_opsets: Collection[OpsetVersion]
+) -> Optional[OpsetVersion]:
+    """Finds the registered opset given a target opset version and the available opsets.
+
+    Args:
+        target: The target opset version.
+        available_opsets: The available opsets.
+
+    Returns:
+        The registered opset version.
+    """
+    if not available_opsets:
+        return None
+    available_versions = sorted(available_opsets)
```

Comment:
```suggest
available_versions = sorted(available_opsets, reverse=True)
```
It makes the following code easier and make more sense?

Reply:
Sure, but we do a linear search in order and reversed. So `reversed` will still need to be called on line 39

Reply:
I will leave it as it if it sounds good to you.

Reply:
Mark as resolved